packages/blade/src/components/Button/BaseButton/BaseButton.tsx

```diff
@@ -28,6 +28,7 @@ import { BaseSpinner } from '~components/Spinner/BaseSpinner';
 import Box from '~components/Box';
 import type { DotNotationSpacingStringToken } from '~src/_helpers/types';
 import type { BaseTextProps } from '~components/Typography/BaseText/types';
+import iconMap from '~components/Icons/iconMap';
```

Comment:
Although this might make the check work it isn't an ideal solution because it would bloat the bundle size with all the imported icons in `iconMap.ts` module.